backtracking/coloring.py

```diff
@@ -0,0 +1,118 @@
+from typing import List, Tuple, Union
+
+"""
```

Comment:
Please move this docstring above the imports.

Reply:
done :+1: